packages/editor/src/store/utils/notice-builder.js

```diff
@@ -89,15 +89,31 @@ export function getNotificationArgumentsForSaveFail( data ) {
 
 	const publishStatus = [ 'publish', 'private', 'future' ];
 	const isPublished = publishStatus.indexOf( post.status ) !== -1;
-	// If the post was being published, we show the corresponding publish error message
-	// Unless we publish an "updating failed" message.
+
+	if ( error.code === 'offline_error' ) {
+		const messages = {
+			publish: __( 'Publishing failed because you were offline.' ),
+			private: __( 'Publishing failed because you were offline.' ),
+			future: __( 'Scheduling failed because you were offline.' ),
+			default: __( 'Updating failed because you were offline.' ),
```

Comment:
I wonder if we can move "You were offline." to a separate string that we concat with the existing "publishing/updating/scheduling failed" string. That way translators will have less to translate. WDYT?

Reply:
@swissspidy or @SergeyBiryukov might have a more definitive answer. I don't remember any exceptions to the rule, but they usually havea  couple 😄